### PR TITLE
Drop support php 7, add php 8.2 support

### PR DESCRIPTION
**Pull request by fezfez**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,16 +18,13 @@
     },
     "config": {
         "sort-packages": true,
-        "platform": {
-            "php": "7.4.99"
-        },
```
Comment (Ocramius) [inline composer.json]: This should stay, but needs updating.

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (fezfez) [reply composer.json]: @Ocramius should be 8.0.99 ?

Comment (Ocramius) [reply composer.json]: Yep, and remember to do at least a `composer update nothing` for the lockfile 👍

Comment (fezfez) [reply composer.json]: done

**Review by Ocramius — APPROVED** [review]
Thanks @fezfez !

**Pull request merged**